test/models/private_tests/test_maskrcnn.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+import onnx
+import pytest
+
+from onnx2kerastl import onnx_to_keras
+import tensorflow as tf
+from keras_data_format_converter import convert_channels_first_to_last
+from test.models.private_tests.aws_utils import aws_s3_download
+
+
+def find_best_matches(A, B):
+    n = A.shape[0]
+    best_matches = np.ones(n, dtype=int)*(-1)
+    for i in range(n):
+        best_distance = float('inf')
+        best_index = -1
+        for j in range(n):
+            if j not in best_matches:
+                distance = np.sum((A[i] - B[j]) ** 2)
+                if distance < best_distance:
+                    best_distance = distance
+                    best_index = j
+        if best_index == -1:
+            raise Exception("Could not greedily match the order of the two arrays")
+        best_matches[i] = best_index
+    return best_matches
```

Comment:
Not very important but this function can be written as:
```
def find_best_matches_vectorized(A, B):
    # Calculate the pairwise Euclidean distances between rows of A and B
    distances = np.sum((A[:, np.newaxis, :] - B[np.newaxis, :, :]) ** 2, axis=2)
    best_matches = np.argmin(distances, axis=1)
    return best_matches
 ```

Reply:
This is not exactly the same :) - I want a greedy algorithm that maps at most one element to every element in B iteratively.
The suggestion will allow multiple mapping to the same index of B